modules/lvms-creating-lvmcluster-using-rhacm.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-lvmcluster-using-rhacm_{context}"]
+= Creating an LVMCluster CR by using {rh-rhacm}
+
+After you have installed {lvms} by using {rh-rhacm}, you must create an `LVMCluster` custom resource (CR).
+
+.Prerequisites
+
+* You have installed {lvms} by using {rh-rhacm}.
+* You have access to the {rh-rhacm} cluster using an account with `cluster-admin` permissions.
+* You read the _About the LVMCluster custom resource_ section. See the _Additional resources_ section.
+
+.Procedure
+
+. Log in to the {rh-rhacm} CLI using your {product-title} credentials.
+
+. Create an `LVMCluster` CR YAML file:
++
+.Example `LVMCluster` CR
+[source, yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+      kind: ConfigurationPolicy
+      metadata:
+        name: lvms
+      spec:
+        object-templates:
+           - complianceType: musthave
+             objectDefinition:
+               apiVersion: lvm.topolvm.io/v1alpha1
+               kind: LVMCluster
+               metadata:
+                 name: my-lvmcluster
+                 namespace: openshift-storage
+               spec:
+                 storage:
+                   deviceClasses: <1>
+                   - name: vg1
+                     default: true
+                     deviceSelector: <2>
+                       paths:
+                       - /dev/disk/by-path/pci-0000:87:00.0-nvme-1
+                       - /dev/disk/by-path/pci-0000:88:00.0-nvme-1
+                       optionalPaths:
+                       - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
+                       - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
+                     thinPoolConfig: <3>
+                       name: thin-pool-1
+                       sizePercent: 90
+                       overprovisionRatio: 10
+                     nodeSelector: <4>
+                       nodeSelectorTerms:
+                       - matchExpressions:
+                           - key: app
+                             operator: In
+                             values:
+                             - test1
+        remediationAction: enforce
+        severity: low
+----
+<1> Contains the configuration to assign the local storage devices to the LVM volume groups.
+<2> Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group, and force wipe the devices that are added to the LVM volume group. 
+<3> Contains the LVM thin pool configuration.
+<4> Contains the configuration to choose the nodes on which you want to create the LVM volume groups. If this field is empty, then all nodes without no-schedule taints are considered.
+
+. Create the `LVMCluster` CR by running the following command:
++
+[source,terminal]
+----
+# oc create -f <file_name> -n <namespace>
```

Comment:
```suggestion
# oc create -f <file_name> -n <ocp-cluster-namespace>
```

Reply:
`ocp-cluster-namespace` is the namespace of the ocp-cluster which was selected to install lvms operator

Reply:
Updated as per the suggestion.